src/networkrequest.cpp

```diff
@@ -2,27 +2,30 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "networkrequest.h"
-#include "captiveportal/captiveportal.h"
-#include "constants.h"
-#include "hawkauth.h"
-#include "leakdetector.h"
-#include "logger.h"
-#include "mozillavpn.h"
-#include "networkmanager.h"
-#include "settingsholder.h"
-#include "task.h"
-
-#include <QDirIterator>
-#include <QHostAddress>
-#include <QJsonDocument>
-#include <QJsonArray>
-#include <QJsonObject>
-#include <QNetworkAccessManager>
-#include <QNetworkReply>
-#include <QNetworkRequest>
-#include <QRegularExpression>
-#include <QUrl>
+// Wasm has its own networkrequest implementation
+#ifndef MVPN_WASM
```

Comment:
I have a better solution. Since this is a one-off quirk for WASM, set the `HEADER_FILE_ONLY` property on `networkrequest.cpp` in `src/cmake/wasm.cmake` which should disable compilation of this file in just this one case.